cmd/kubeadm/app/apis/kubeadm/v1alpha2/upgrade_test.go

```diff
@@ -0,0 +1,137 @@
+/*
```

Comment:
probably don't need to copy this over? or if we do, we should delete it from v1alpha1

Reply:
oh, I will delete this. This isn't needed in v1alpha2